.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file is related to your IDE, it should not be part from the project
so please add it to `.gitignore`